.github/workflows/golang-release.yml

```
@@ -0,0 +1,69 @@
+name: Go Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_number:
+        description: "New Version"
+        required: true
+
+env:
+  GO111MODULE: on
+  GO_VERSION: "^1.18"
+
+jobs:
+  build_rust:
+    uses: ./.github/workflows/build-rust-cross-platform.yml
+
+  generate-schemas:
+    uses: ./.github/workflows/generate_schemas.yml
+
+  build:
+    needs: [build_rust, generate-schemas]
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```
We typically pin runner versions to a specific version, Ubuntu 22.04 can be used here.

---

changed